sources/processors/wikipedia/rank.py

```diff
@@ -112,10 +120,23 @@ def football_stadia_by_size(page, wikidata):
         is_stadium = claim_watch("P31",        #instance of
                                  "Q1154710",   #football stadium
                                  wikidata=wikidata)
-        max_capacity = get_quantity("P1083",  #maximum capacity
+        max_capacity = get_quantity("P1083",   #maximum capacity
                                     wikidata=wikidata)
         return is_stadium * max_capacity
-    
+
+    @staticmethod
+    def whats_on_tv(page, wikidata):
+        is_on_tv = claim_exists("P3301",       #broadcast by
+                                 wikidata=wikidata)
+        event_date = get_time("P585",          #point in time
+                              wikidata=wikidata)
+        day_diff = (event_date - datetime.date.today()).days
+        
+        if day_diff < 0:
+            return 0
+        else:
+            return 100 * is_on_tv / (0.2 + day_diff)
```

Comment:
I'm wondering about why your choosing the 100 and 0.2 values. Why not use it like whats_on_tv=100 instead? Do you want to prevent division by 0 with the +0.2? I'd like to understand so I could update it if I ever need to.

Reply:
Yes, the 100 is a bit pointless -- I was writing it while you were doing the demo and it looked like about the right scale. But I know it's not really the right place for it.

The "+ 0.2" is indeed preventing division by zero, and to pick the number I gave just a few seconds' thought to what it means for the relative ranks of something today/tomorrow/a few days time; but you could easily use a completely different function for weighting by how many days away it is.

Reply:
Thanks for clarifying. Merging now. Deploying later today (a class is using it today, no idea at what time exactly). I'll ping you when you can use it in a feed. Thanks again!